src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -419,11 +409,21 @@
                      :transform [{:translate-y content-translate-y}]}
                     {})}
            (case @active-tab
-             1 [scan-qr-code-tab qr-view-finder request-camera-permission]
+             1 [scan-qr-code-tab qr-view-finder]
              2 [enter-sync-code-tab]
              nil)]
           [rn/view {:style style/flex-spacer}]
-          (when show-bottom-view? [bottom-view insets bottom-view-translate-y])]]))))
+          (when show-bottom-view? [bottom-view insets bottom-view-translate-y])
+          (when (and (or (not animated?) @render-camera?) show-camera?)
+            [quo/button
+             {:icon                      true
+              :type                      :blur-bg
+              :size                      style/flash-button-size
+              :accessibility-label       :camera-flash
+              :override-background-color colors/neutral-80-opa-40
```

Comment:
If possible remove the use of 'override-background-color' prop. It was added initially to handle the customisation color but has just ended up with us all not using the design system correctly - this should be handled by the type etc. the correct color should already be set within the button.

Also if the current configuration doesn't work without removing that, it's okay as my refactor should fix this and I can address in a follow up.

Reply:
According to Figma, we have to use type `grey` and background `photo`. We can update this in a follow-up once we merge the PR for the button refactor. For now, I can use the `photo-bg` type to remove the `override-background-color` prop. wydt?

Reply:
I think let's just leave it as is because both will be cleaned up :)